x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/AnalyzerTests.java

```diff
@@ -1554,6 +1554,97 @@ public void testUnresolvedMvExpand() {
         assertThat(e.getMessage(), containsString("Unknown column [bar]"));
     }
 
+    public void testLiteralInAggregateNoGrouping() {
+        var e = expectThrows(VerificationException.class, () -> analyze("""
+             from test
+            |stats 1
```

Comment:
I prefer to have separate tests since in case of a failure it is to identify what case failed based on the method name:
`testLiteralBehindEvalInAggregateNoGrouping`.
Tests are one area where I prefer to be verbose and thus kind to my future self/developer who might investigate what's going on.